src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
+
+  if (days < 3) {
+    result = days * 40;
+  }
+
+  if (days < 7 && days > 2) {
+    result = (days * 40) - 20;
+  }
+
+  if (days >= 7) {
+    result = (days * 40) - 50;
+  }
+
+  return result;
```

Comment:
Let's move numbers to variables so it's understandable what they represent. Also, you can return result right away when you go in if condition, as if days are less than 3 there's no need to go further and check other conditions.